packages/types/src/sdkinfo.ts

```diff
@@ -1,6 +1,9 @@
 import { Package } from './package';
 
-/** JSDoc */
+/**
+ * TODO: Remove `applyIntegrationsMetadata` method from the core and rely on the API instead, once we move to envelopes for all types of requests.
```

Comment:
I'm not clear why this has to wait for error events to be sent in envelopes. If all SDK metadata is set on the `Api` object when `init` runs (see note above), then we can just read from there when we add SDK data to the event (which we'll still need to at least partially do, even when envelopes are a universal thing, because the envelope header won't contain `integrations` or `packages`).

Reply:
Solved.